sdc/tests/tests_perf/test_perf_utils.py

```diff
@@ -195,13 +195,78 @@ def get_times(f, *args, iter_number=5):
     return exec_times, boxing_times
 
 
+class ResultsDriver:
+    """Base class. Load and dump results."""
+
+    def __init__(self, file_name, raw_file_name=None):
+        self.file_name = file_name
+        self.raw_file_name = raw_file_name if raw_file_name else f'raw_{file_name}'
+
+
+class ExcelResultsDriver(ResultsDriver):
+    # openpyxl need to be installed
+
+    def dump_grouped_data(self, grouped_data, logger=None):
```

Comment:
Would we like to make `logger` as object attribute, maybe of base class (`self.logger`)?

Reply:
Driver is created before creating TestResults. Logger is created inside TestResults. I think working with logger should be redesigned completely. Not in this PR.